docker/python3-deb/Dockerfile

```
@@ -28,5 +28,12 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   python3-dev \
 && rm -rf /var/lib/apt/lists/*
 
+
+RUN cp /etc/ssl/openssl.cnf /etc/ssl/openssl.cnf.org && \
```

Maybe add a timestamp to the back filename?
```suggestion
RUN cp /etc/ssl/openssl.cnf "/etc/ssl/openssl_$(date +"%Y%m%d_%H%M").cnf" && \
```